### PR TITLE
sim_vehicle.py : coding style correction

### PR DESCRIPTION
**Pull request by khancyr**
using pep8 and pylint

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
This needs a rebase and some correction. Could you do the rework and resubmit?

  - Commit messages need a blank line between the first line and the body
  - Last commit should actually be squashed with previous commit - actually sts and pid are used and another fix has already landed in master

**Comment by khancyr** [comment]
It should be correct now !

**Comment by khancyr** [comment]
By the way, should we initiate a coding style for python ? 
Using pep8 or pylint (without line lenght check) should not be too restrictive

File: Tools/autotest/sim_vehicle.py [hunk 1]
```diff
@@ -121,95 +130,97 @@ def kill_tasks():
...
     jsbsim_cmd = ["JSBSim", "--version"]
     progress_cmd("Get JSBSim version", jsbsim_cmd)
     try:
         jsbsim_version = subprocess.Popen(jsbsim_cmd, stdout=subprocess.PIPE).communicate()[0]
     except OSError as e:
-        jsbsim_version = '' # this value will trigger the ".index"
-                            # check below and produce a reasonable
-                            # error message
+        jsbsim_version = ''  # this value will trigger the ".index"
+        # check below and produce a reasonable
```
Comment (peterbarker) [inline Tools/autotest/sim_vehicle.py]: This indenting isn't so great.  The tabs in the original were wrong, but I think the concept is right (which obviously puts me at odds with the tool used to create the diff...

**Comment by peterbarker** [comment]
I'm in favour of some sort of standard.  I think we should determine what that is before merging this one in, just in case.  @tridge are you OK with the proposed Python coding style?

I'm also not sure what to think about these:
``Rename Invalid argument name "procName" (invalid-name)``
``Rename Invalid variable name "outputLines" (invalid-name)``
``Rename Invalid variable name "lineSplit" (invalid-name)``

Those names were carried over from the source for that code - and that source is referenced.  Changing the variable names seems... odd to me.

File: Tools/autotest/sim_vehicle.py (hunk 1 above)
Comment (khancyr) [inline Tools/autotest/sim_vehicle.py]: I agree that it seems less clear ... sometimes we need to break rules I think

**Comment by khancyr** [comment]
Ok, let's wait for a coding standard before merging this.
Should we create an issue or a post on discourse ? I propose PEP-8 because many tools using it but there could be better codestyle (with associated tool)

**Comment by lucasdemarchi** [comment]
pep8 is a good option, but should be done again rather than just rebasing this commit since lots of stuff changed there. I'm closing this for now.

**Pull request closed without merging**